### PR TITLE
Adding lombok asset generation in java11 and java11-openj9

### PR DESCRIPTION
**Pull request by ghatwala**
1. During "plugin-java11" and "plugin-java11--openj9" image builds creating lombok assets.
2. Here check exists already if lombok latest exists or not - https://github.com/redhat-developer/codeready-workspaces-images/blob/crw-2-rhel-8/codeready-workspaces-plugin-java8/build/build_lombok.sh#L53-#L57

signed-off-by : Amit Ghatwal <ghatwala@us.ibm.com>

### CODE REVIEW

File: codeready-workspaces-plugin-java11/get-sources.sh [hunk 1]
```diff
@@ -27,6 +27,23 @@ function log()
...
+
+if [[ ${DELETE_ASSETS} -eq 1 ]]; then
+	log "[INFO] Delete Previous GitHub Releases:"
+	./uploadAssetsToGHRelease.sh --delete-assets -v "${CSV_VERSION}" -n ${ASSET_NAME}
+	exit 0;
+fi
+
+if [[ ${PUBLISH_ASSETS} -eq 1 ]]; then
+	log "[INFO] Build Assets and Publish to GitHub Releases:"
+	./build/build.sh -v ${CSV_VERSION} -n ${ASSET_NAME}
```
[the remark this answers is not in the document]
Comment (ghatwala) [reply codeready-workspaces-plugin-java11/get-sources.sh:43]: had kept this to keep it consistent with what's done in plugin-java8's - `get -sources.sh` - https://github.com/redhat-developer/codeready-workspaces-images/blob/crw-2-rhel-8/codeready-workspaces-plugin-java8/get-sources.sh#L51 and plugin-java8-openJ9's - `get-sources.sh` - https://github.com/redhat-developer/codeready-workspaces-images/blob/crw-2-rhel-8/codeready-workspaces-plugin-java8-openj9/get-sources.sh#L53 ....

File: codeready-workspaces-plugin-java11-openj9/build/build.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -xe
+
+# Copyright (c) 2018-2021 Red Hat, Inc.
```
[the remark this answers is not in the document]
Comment (ghatwala) [reply codeready-workspaces-plugin-java11-openj9/build/build.sh:3]: done

File: codeready-workspaces-plugin-java11/build/build.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -xe
+
+# Copyright (c) 2018-2021 Red Hat, Inc.
```
[the remark this answers is not in the document]
Comment (ghatwala) [reply codeready-workspaces-plugin-java11/build/build.sh:3]: done

File: codeready-workspaces-plugin-java11-openj9/get-sources.sh [hunk 4]
```diff
@@ -29,6 +29,23 @@ function log()
 	fi
 }
 
+
+if [[ ! -x ./uploadAssetsToGHRelease.sh ]]; then 
+    curl -sSLO "https://raw.githubusercontent.com/redhat-developer/codeready-workspaces/${MIDSTM_BRANCH}/product/uploadAssetsToGHRelease.sh" && chmod +x uploadAssetsToGHRelease.sh
+fi
+
+if [[ ${DELETE_ASSETS} -eq 1 ]]; then
```
Comment (nickboldt) [inline codeready-workspaces-plugin-java11-openj9/get-sources.sh:37]: if you're going to add code that's triggered by `DELETE_ASSETS` ... you need to set that variable up on line 16. Same for `PUBLISH_ASSETS`. See other get-sources.sh scripts for inspiration.

**Review by nickboldt — APPROVED** [review]

**Pull request merged**